### ISSUE TITLE
Jetpack Dashboard: Headings skewed when using 1496 x 967 resolution on MacBook Pro

### ISSUE THREAD

**Issue by jenhooks** [Automattic/jetpack#31951]
### Impacted plugin

Jetpack

### Quick summary

When using a scaled screen resolution (1496 x 967) on a 2023 16-inch MacBook Pro, the headings in Jetpack's Dashboard are skewed left in Chrome. 

<img width="866" alt="Screenshot 2023-07-18 at 4 54 52 PM" src="https://github.com/Automattic/jetpack/assets/7585054/7cfbe5b6-0775-4c89-a46b-6238e29428bb">

The spacing problems do not persist in other scaled resolutions, according to my testing. I reproduced this on a JN test site, a Pressable test site, and a SiteGround site. 

This will likely only impact users who have their resolution on a MacBook set to 1496 x 967, though there may be other resolutions on other devices similarly impacted.

### Steps to reproduce

1. Install and connect Jetpack on a WordPress site
2. Navigate to Jetpack → Dashboard
3. Note orientation of the Menu (At a Glance, My Plan, Plans, etc.) and the Headings (Jetpack Stats, Security, etc) , note proper spacing to the left. 
4. Navigate to System Settings → Display in Mac
5. Change resolution to 1496 x 967
6. Note that the orientation of the Menu and Headings skews left. 

### A clear and concise description of what you expected to happen.

I expect the spacing to persist regardless of screen resolution.

### What actually happened

The space between the Menu and Headings on Jetpack → Dashboard disappears.

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

**Comment by andrii-lysenko** [comment]
I wasn't able to reproduce the issue on the mentioned resolution. I was also trying other resolutions as well as manual resizing and could spot anything like this. There are other teams working on that page, so it might be that someone has already fixed that.

@jenhooks could you please verify if the issue is still present? Thank you!

**Comment by jenhooks** [comment]
I'm not able to reproduce this any more, thanks for the ping though, @andrii-lysenko

**Comment by andrii-lysenko** [comment]
Thank you. I think this should be safe to close. We could open a new one if the issue appears once again.
